### DISCUSSION TITLE
How to use output "EMS_START_MANUAL_CHARGE" of request "EMS_REQ_START_MANUAL_CHARGE"?

### DISCUSSION THREAD

**Discussion by hmichels** [spali/go-rscp#49] [Q&A]
Hello Spali,

Once again, your tool is great and helps me a lot, thank you very much!

I start manual battery charging via the grid using the service 
`service: shell_command.e3dc_request`
`data_template:`
`  request: >-`
`    {{ [["EMS_REQ_START_MANUAL_CHARGE", 250]] }}`
and get the following output/answer: 
`Tue Nov 28 11:47:03 CET 2023: [["EMS_REQ_START_MANUAL_CHARGE", 250]]{"EMS_START_MANUAL_CHARGE":false}`

Info: I know that this request can only be executed once every hour, therefore the result here is "false".

Questions: 
How can I use/receive the output **{"EMS_START_MANUAL_CHARGE":false}** ?
Do I need another script, command_line or template sensor for this?

An example would be nice ;-)

Many thanks and best regards
Holger

**Comment by spali** [comment]
Never use the new functionality of recent hass. But check chapter "Response" of https://www.home-assistant.io/integrations/shell_command/
